src/14_cal.py

```diff
@@ -21,4 +21,17 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
```

Comment:
This works, but I don't think it is specifically what the spec is asking for. In you version there are required inputs and in the spec it is looking for different things to happen with different inputs. Try this code 
```
l = len(sys.argv)

if l == 1:
    # User didn't specify any input
    month = datetime.now().month
    year = datetime.now().year
elif l == 2:
    # User didn't specify year
    month = int(sys.argv[1])
    year = datetime.now().year
elif l == 3:
    # User specified both month and year
    month = int(sys.argv[1])
    year = int(sys.argv[2])
else:
    # User provided faulty input
    print("usage: calendar.py [month] [year]")
    sys.exit(1)

cal = calendar.TextCalendar()

cal.prmonth(year, month)
```
Then run in the command line with arguments
`python3 14_cal.py 09 2011`
or no arguments
`python3 14_cal.py`